source/user-guide/how-to-update-pagerduty-status-page.html.md.erb

```diff
@@ -0,0 +1,32 @@
+---
+owner_slack: "#modernisation-platform"
+title: How to update the pagerduty status page in the Modernisation Platform
+last_reviewed_on: 2023-11-02
+review_in: 6 months
+---
+
+# <%= current_page.data.title %>
+
+## Introduction
+If there is an incident or planned you will need to update the external status page for the modernisation platform [modernisation platform status page](https://status.modernisation-platform.service.justice.gov.uk/) this guide will show you how to do this.
```

Comment:
or planned... outage?

Reply:
there is a missing word maintenance should be there

Reply:
new commit added